File: rafael/arrays/SequenciaCrescente.java
class SequenciaCrescente {
	public static void main(String[] args) {
		int[] array = new int[10];

		for (int i = 0; i < array.length; i++) {
			array[i] = i;
		}

		for (int i = 0; i < array.length; i++) {
			System.out.println(array[i]);
		}
	}
}

File: rafael/arrays/SequenciaCrescente.cs
class SequenciaCrescente 
{
	static void Main() 
	{
		int[] array = new int[10];

		for (int i = 0; i < array.Length; i++) 
		{
			array[i] = i;
		}

		for (int i = 0; i < array.Length; i++) 
		{
			System.Console.WriteLine(array[i]);
		}
	}
}
